### PR TITLE
DBot predict phishing words enhancement

### PR DESCRIPTION
**Pull request by erezh31**
## Status
Ready

### CODE REVIEW

**Comment by erezh31** [comment]
@kirbles19 plz review docs
@Itay4 plz review

File: Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 1]
```diff
@@ -1,27 +1,36 @@
 # pylint: disable=no-member
-import demistomock as demisto
-from CommonServerPython import *
-from CommonServerUserPython import *
-
 import demisto_ml
 
+from CommonServerPython import *
```
Comment (Itay4) [inline Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py]: please move back to top of the file

**Comment by erezh31** [comment]
@richardbluestone plz review docs, thanks!

**Comment by erezh31** [comment]
@roysagi plz review

File: Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 2]
```diff
@@ -34,13 +42,17 @@ def predict_phishing_words(model_name, model_store_type, email_subject, email_bo
...
     explain_result = demisto_ml.explain_model_words(model,
                                                     input_text,
-                                                    float(demisto.args().get('labelProbabilityThreshold', 0)),
-                                                    float(demisto.args().get('wordThreshold', 0)),
-                                                    int(demisto.args()['topWordsLimit']))
+                                                    0,
+                                                    word_threshold,
+                                                    top_word_limit)
+    if explain_result["Probability"] < label_threshold:
+        handle_error("Label probability is %.2f and its below input threshold", is_return_error)
```
Comment (amshamah419) [inline Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py]:
```suggestion
        handle_error("Label probability is %.2f and it's below the input threshold", is_return_error)
```

File: Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml [hunk 3]
```diff
@@ -63,6 +63,17 @@ args:
   name: labelProbabilityThreshold
   required: false
   secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'true'
+  description: Whether to to return error when there is no prediction
```
Comment (amshamah419) [inline Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml]:
```suggestion
  description: Whether to return an error when there is no prediction.
```

**Pull request merged**